<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresupuestoAprobado</class>
 <widget class="QWidget" name="PresupuestoAprobado">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro Presupuesto Aprobado</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/iconos/cosas.jpg</normaloff>:/iconos/cosas.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Variación de Gastos por Unidad Presupuestaria Linea de Trabajo,Rubro, Cuenta y Especifico</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Datos Generales de Variación de Gastos por Unidad  Presupuestaria</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="Institucion">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Institucion</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="EjercicioFiscal">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Ejercicio Fiscal:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="EjercicioFiscal2">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QSplitter" name="splitter_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="EjercicioFiscal_4">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Fuente Financiamiento:</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="FuenteFinanciamiento">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="4">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="FechaElaboracion">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Fecha Elaboracion</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="Fecha">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="Institucion2"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Especificacion Presupuesto Solicitado</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tablaPresupuestoAprobado">
           <property name="minimumSize">
            <size>
             <width>261</width>
             <height>211</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>600</width>
             <height>500</height>
            </size>
           </property>
           <property name="cursor" stdset="0">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::AllEditTriggers</set>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>true</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="textElideMode">
            <enum>Qt::ElideNone</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerItem</enum>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="rowCount">
            <number>0</number>
           </property>
           <property name="columnCount">
            <number>3</number>
           </property>
           <column>
            <property name="text">
             <string>   Codigo Cuenta</string>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string notr="true">Nombre de Cuenta</string>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Presupuesto Aprobado</string>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </column>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Registro: Presupuesto Solicitado</string>
           </property>
           <widget class="QWidget" name="layoutWidget_2">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>90</y>
              <width>111</width>
              <height>61</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="AgregarButton">
               <property name="text">
                <string>Agregar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="cancelarButton">
               <property name="text">
                <string>Cancelar</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/iconos/pre2-iconos.png</normaloff>:/iconos/pre2-iconos.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="1" column="0" rowspan="2">
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Total Supuesto Aprobado</string>
           </property>
           <widget class="QSplitter" name="splitter_2">
            <property name="geometry">
             <rect>
              <x>430</x>
              <y>30</y>
              <width>139</width>
              <height>20</height>
             </rect>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QLineEdit" name="montoTotallLine">
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
           <widget class="QCommandLinkButton" name="TotalAprobadoLinkButton">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>20</y>
              <width>221</width>
              <height>41</height>
             </rect>
            </property>
            <property name="text">
             <string>Obtener Total Aprobado</string>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Opciones</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="salirButton">
              <property name="text">
               <string>Salir</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>salirButton</sender>
   <signal>clicked()</signal>
   <receiver>PresupuestoAprobado</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>761</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
